spring-webhook-validation/src/main/java/com/telnyx/webhookverification/WebhookController.java

```diff
@@ -0,0 +1,47 @@
+package com.telnyx.webhookverification;
+
+import org.bouncycastle.crypto.Signer;
+import org.bouncycastle.crypto.params.Ed25519PublicKeyParameters;
+import org.bouncycastle.crypto.signers.Ed25519Signer;
+import org.bouncycastle.jce.provider.BouncyCastleProvider;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.http.HttpHeaders;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestHeader;
+import org.springframework.web.bind.annotation.RestController;
+import io.github.cdimascio.dotenv.Dotenv;
+import java.nio.charset.StandardCharsets;
+import java.security.Security;
+import java.util.Base64;
+
+@RestController
+public class WebhookController {
+
+    private static final Dotenv dotenv = Dotenv.load();
+    private final String TELNYX_PUBLIC_KEY = dotenv.get("TELNYX_PUBLIC_KEY");
+
+    @PostMapping("/messaging/inbound")
+    public ResponseEntity<Void> webhook(@RequestBody String payload, @RequestHeader HttpHeaders headers) {
+        Security.addProvider(new BouncyCastleProvider());
+        String signature = headers.get("telnyx-signature-ed25519").get(0);
+        String timestamp = headers.get("telnyx-timestamp").get(0);
```

Comment:
Add a `try/catch` to pull these headers and return `HttpStatus.UNAUTHORIZED` if they're not present. 

Will throw a 500-error when pulling errors if not available

Reply:
@drewkitch Add a try/catch to pull these headers and return HttpStatus.UNAUTHORIZED if they're not present.

Will throw a 500-error when pulling errors if not available